src/Shipment/TaxIdentificationNumbers/Enums/TaxNumberTypeEnum.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Shipment\TaxIdentificationNumbers\Enums;
+
+use MyCLabs\Enum\Enum;
+/**
+ * @method static self EORI()
+ * @method static self VAT()
+ * @method static self IOSS()
+ */
+class TaxNumberTypeEnum extends Enum
```

Comment:
I think that `TaxNumberType` is enough as a name. Same style has already been used for ContetType for instance. 
```suggestion
class TaxNumberType extends Enum
```

Reply:
This means our coding style differs per project? What are we doing in new projects? Do we suffix or not? How do I know which to use when, do I always just copy the style others have used in that specific project?

Reply:
Yeah, good point. In the api we have the Enum suffix and it has now been used for too long. Change is not necessarily difficult but then changing it for all projects might be a lot of work. For me not using the suffix is better regardless of the project. However, I think the conceptual integrity of the project should also play a role so that we can keep things organized and not confusing. That's why I'd suggest to follow the project's style, and if you create a new project then you make the decision on what to use. After that all is getting tuned up during PR reviews.